src/components/AttendantForm.test.jsx

```diff
@@ -0,0 +1,151 @@
+import AttendantForm from "./AttendantForm";
+import { render, screen, waitFor } from "@testing-library/react";
+import userEvent from "@testing-library/user-event";
+import ErrorMessages from "../constants/errorMessages";
+
+const jobTitles = ["Accountant", "Engineer", "Lawyer", "Teacher"];
+
+const fillAndSubmitForm = async () => {
+  userEvent.type(screen.getByTestId("name"), "John");
+  userEvent.type(screen.getByTestId("last-name"), "Doe");
+  userEvent.selectOptions(await screen.findByTestId("job-title"), "Engineer");
+  userEvent.type(screen.getByTestId("age"), "40");
+  userEvent.click(screen.getByTestId("submit"));
+};
+
+describe("<AttendantForm />", () => {
+  let props;
+
+  beforeEach(() => {
+    props = {
+      isAttendantsLoading: false,
+      isJobTitlesLoading: false,
+      submitAttendant: jest.fn(),
+      jobTitles: jobTitles,
+      jobTitlesApiError: null,
+    };
+  });
+
+  it("renders correctly when data is loaded", async () => {
+    const { container } = render(<AttendantForm {...props} />);
+
+    expect(screen.queryByTestId("job-titles-loader")).not.toBeInTheDocument();
+    expect(container).toMatchSnapshot();
+  });
+
+  it("renders job title loader", async () => {
+    render(
+      <AttendantForm
+        {...props}
+        isJobTitlesLoading={true}
+      />
+    );
+
+    expect(screen.getByTestId("job-titles-loader")).toBeInTheDocument();
+  });
+
+  it("renders error message when get job titles API call fails", async () => {
+    render(
+      <AttendantForm
+        {...props}
+        jobTitlesApiError={[ErrorMessages.FAILED_TO_GET_JOB_TITLES]}
+      />
+    );
+
+    expect(
+      screen.getByText(ErrorMessages.FAILED_TO_GET_JOB_TITLES)
```

Comment:
For the assertions use actual text. It is not right/wrong to do it your way, in general in vinted literal text is asserted